# Occitan translation of Pybliographer.
# Copyright (C) 2003-2006 Free Software Foundation, Inc.
# This file is distributed under the same license as the pybliographer package.
#
# Yannig MARCHEGAY (yannig@marchegay.org) - 2006-2007
#
msgid ""
msgstr ""
"Project-Id-Version: pybliographer-1.1.93\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2007-05-01 10:57+0200\n"
"PO-Revision-Date: 2007-08-07 14:08+0200\n"
"Last-Translator: Yannig MARCHEGAY (Kokoyaya) <yannig@marchegay.org>\n"
"Language-Team: Occitan (post 1500) <ubuntu-l10n-oci@lists.ubuntu.com>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);"

#: ../Pyblio/Autoload.py:78
#, python-format
msgid "warning: can't import %s: %s"
msgstr ""

#: ../Pyblio/Config.py:52
#, python-format
msgid "value of `%s' should be of type %s"
msgstr ""

#: ../Pyblio/Config.py:212
msgid "String"
msgstr "Cadena"

#: ../Pyblio/Config.py:221
msgid "Boolean"
msgstr ""

#: ../Pyblio/Config.py:240
msgid "Integer"
msgstr "Entièr"

#: ../Pyblio/Config.py:242
#, python-format
msgid "Integer under %d"
msgstr ""

#: ../Pyblio/Config.py:244
#, python-format
msgid "Integer over %d"
msgstr ""

#: ../Pyblio/Config.py:246
#, python-format
msgid "Integer between %d and %d"
msgstr ""

#: ../Pyblio/Config.py:258
#, python-format
msgid "Element in `%s'"
msgstr ""

#: ../Pyblio/Config.py:278
#, python-format
msgid "Tuple (%s)"
msgstr ""

#: ../Pyblio/Config.py:299
#, python-format
msgid "List (%s)"
msgstr ""

#: ../Pyblio/Config.py:324
#, python-format
msgid "Dictionary (%s, %s)"
msgstr ""

#: ../Pyblio/Config.py:344
#, python-format
msgid "warning: could not restore setting %r to %r: %s"
msgstr ""

#: ../Pyblio/Fields.py:261
msgid "Illegal year value"
msgstr ""

#: ../Pyblio/Fields.py:265
msgid "Illegal month value"
msgstr ""

#: ../Pyblio/Fields.py:269
msgid "Illegal day value"
msgstr ""

#: ../Pyblio/Fields.py:510
#, python-format
msgid "in %s"
msgstr ""

#: ../Pyblio/Format/BibTeX.py:385 ../Pyblio/Format/BibTeX.py:443
#, python-format
msgid "key `%s' is malformed"
msgstr ""

#: ../Pyblio/Format/BibTeX.py:406 ../Pyblio/Format/BibTeX.py:427
#, python-format
msgid "%s:%d: key `%s' already defined"
msgstr ""

#: ../Pyblio/GnomeUI/Config.py:147 ../Pyblio/GnomeUI/Fields.py:417
msgid ""
"Some changes require to restart Pybliographic\n"
"to be correctly taken into account"
msgstr ""

#: ../Pyblio/GnomeUI/Config.py:429
msgid "Add"
msgstr "Apondre"

#: ../Pyblio/GnomeUI/Config.py:432
msgid "Update"
msgstr "Actualizar"

#: ../Pyblio/GnomeUI/Config.py:435 ../Pyblio/GnomeUI/Config.py:549
msgid "Remove"
msgstr "Suprimir"

#: ../Pyblio/GnomeUI/Config.py:546
msgid "Set"
msgstr "Definir"

#: ../Pyblio/GnomeUI/Config.py:561
msgid "Key:"
msgstr "Clau :"

#: ../Pyblio/GnomeUI/Config.py:563
msgid "Value:"
msgstr "Valor :"

#. id     stock            label         accel   tooltip   callback
#: ../Pyblio/GnomeUI/Document.py:145
msgid "_File"
msgstr "_Fichièr"

#: ../Pyblio/GnomeUI/Document.py:146 ../Pyblio/GnomeUI/Document.py:195
msgid "_Edit"
msgstr "_Edicion"

#: ../Pyblio/GnomeUI/Document.py:147
msgid "_View"
msgstr "_Mostrar"

#: ../Pyblio/GnomeUI/Document.py:148
msgid "_Cite"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:149
msgid "_Settings"
msgstr "_Paramètres"

#: ../Pyblio/GnomeUI/Document.py:150
msgid "_Help"
msgstr "_Ajuda"

#: ../Pyblio/GnomeUI/Document.py:151
msgid "Recent documents"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:154
msgid "Open a file"
msgstr "Dobrir un fichièr"

#: ../Pyblio/GnomeUI/Document.py:155
msgid "Open _Location"
msgstr "Dobrir un _emplaçament"

#: ../Pyblio/GnomeUI/Document.py:156
msgid "Save the current file"
msgstr "Enregistrar lo fichièr actiu"

#: ../Pyblio/GnomeUI/Document.py:161
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:8
msgid "Merge With..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:162
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:7
msgid "Medline Query..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:170
msgid "Add a new entry"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:174
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:12
msgid "S_ort..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:175
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:2
msgid "Cite..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:175
msgid "Cite key(s)"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:176
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:6
msgid "Format..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:178
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:4
msgid "Fields..."
msgstr "~Camps..."

#: ../Pyblio/GnomeUI/Document.py:180
#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:5
msgid "Forget all changes"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:183
msgid "_Resource"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:194
msgid "_About"
msgstr "_A prepaus"

#: ../Pyblio/GnomeUI/Document.py:242
msgid "Quick search"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:316
msgid "Unnamed bibliographic database"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:399
#, python-format
msgid ""
"can't open file `%s' for writing:\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:420
#, python-format
msgid ""
"Error while parsing `%s':\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:436
msgid "New database"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:444 ../Pyblio/GnomeUI/Document.py:448
msgid "[no entry]"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:445
msgid "[1 entry]"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:446
#, python-format
msgid "[%d entries]"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:449
#, python-format
msgid "[%d/1 entry]"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:450
#, python-format
msgid "[%d/%d entries]"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:455
msgid "[modified]"
msgstr "[modificat]"

#: ../Pyblio/GnomeUI/Document.py:458
#, python-format
msgid "view limited to: %s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:471
msgid ""
"The database has been modified.\n"
"Discard changes?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:475
msgid ""
"The database has been modified.\n"
"Save changes?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:492
msgid ""
"The database has been externally modified.\n"
"Overwrite changes ?"
msgstr ""

#. no result.
#: ../Pyblio/GnomeUI/Document.py:521
msgid "Your query returned no result"
msgstr ""

#. get a new file name
#: ../Pyblio/GnomeUI/Document.py:531
msgid "Merge file"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:543 ../Pyblio/GnomeUI/Document.py:629
msgid "Open error"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:567
msgid "Merge status"
msgstr ""

#. get a new file name
#: ../Pyblio/GnomeUI/Document.py:578
msgid "Open file"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:616
msgid ""
"An autosave file was found which is newer than the original file.\n"
"Do you want to restore it?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:650 ../Pyblio/GnomeUI/Document.py:791
#: ../Pyblio/GnomeUI/Document.py:851
#, python-format
msgid ""
"Unable to remove autosave file `%s':\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:701
#, python-format
msgid ""
"Error during autosaving:\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:729
#, python-format
msgid ""
"Unable to save `%s':\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:737
msgid ""
"An internal error occured during saving\n"
"Try to Save As..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:752
msgid "Save As..."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:758
#, python-format
msgid ""
"The file `%s' already exists.\n"
"Overwrite it ?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:765
#, python-format
msgid ""
"During opening:\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:803
msgid "Reopen error"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:876
#, python-format
msgid ""
"An entry called `%s' already exists.\n"
"Rename and add it anyway ?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:913
msgid "Really remove all the entries ?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:933
msgid "Create new entry"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:947
#, python-format
msgid "Really edit %d entries ?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:993
#, python-format
msgid "Remove all the %d entries ?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:995
#, python-format
msgid "Remove entry `%s' ?"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1014
msgid ""
"your search text must contain\n"
"latin-1 characters only"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1093 ../Pyblio/GnomeUI/Document.py:1101
#, python-format
msgid ""
"Can't connect to LyX:\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1216
#, python-format
msgid ""
"Can't display documentation:\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1224
msgid "This program is copyrighted under the GNU GPL"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1225
msgid "GNOME interface to the Pybliographer system."
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1236
msgid "GNOME Translation Team"
msgstr ""

#: ../Pyblio/GnomeUI/Document.py:1241
msgid "Pybliographer Home Page"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:35
msgid "Last Name, First Name"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:36 ../Pyblio/GnomeUI/Editor.py:1327
#: ../Pyblio/GnomeUI/Fields.py:39 ../Pyblio/GnomeUI/glade/config1.glade.h:14
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:19
msgid "Text"
msgstr "Tèxt"

#: ../Pyblio/GnomeUI/Editor.py:133 ../Pyblio/GnomeUI/Editor.py:1010
msgid "Error in native string parsing"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:312
msgid "Day"
msgstr "Jorn"

#: ../Pyblio/GnomeUI/Editor.py:321
msgid "Month"
msgstr "Mes"

#: ../Pyblio/GnomeUI/Editor.py:330
msgid "Year"
msgstr "Annada"

#: ../Pyblio/GnomeUI/Editor.py:355
msgid "Invalid day field in date"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:363
msgid "Invalid month field in date"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:371
msgid "Invalid year field in date"
msgstr ""

#. A delete button
#: ../Pyblio/GnomeUI/Editor.py:419 ../Pyblio/GnomeUI/Search.py:86
msgid "Delete"
msgstr "Suprimir"

#: ../Pyblio/GnomeUI/Editor.py:430 ../Pyblio/GnomeUI/Editor.py:443
#: ../Pyblio/GnomeUI/Editor.py:446
msgid "[ Drop an Entry here ]"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:502
msgid "Browse..."
msgstr "Examinar..."

#: ../Pyblio/GnomeUI/Editor.py:525
msgid "Select file"
msgstr "Seleccionatz un fichièr"

#: ../Pyblio/GnomeUI/Editor.py:549 ../Pyblio/GnomeUI/Fields.py:236
#: ../Pyblio/GnomeUI/glade/config1.glade.h:8
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:6
msgid "Entry type"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:551
msgid "Key"
msgstr "Clau"

#: ../Pyblio/GnomeUI/Editor.py:600
msgid "Create Field"
msgstr ""

#. navigation buttons
#: ../Pyblio/GnomeUI/Editor.py:605
msgid "Back"
msgstr "Precedent"

#: ../Pyblio/GnomeUI/Editor.py:608
msgid "Next"
msgstr "Seguent"

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Fields.py:451
msgid "Mandatory"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Fields.py:452
msgid "Optional"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:728 ../Pyblio/GnomeUI/Editor.py:1242
msgid "Notes"
msgstr "Nòtas"

#: ../Pyblio/GnomeUI/Editor.py:728
msgid "Extra"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:907
msgid "Invalid key format"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:915
#, python-format
msgid "Key `%s' already exists"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:932
#, python-format
msgid "The `%s' field contains a non Latin-1 symbol"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1002
msgid "Your text contains non Latin-1 symbols"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1023
msgid "Edit entry"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1037 ../Pyblio/GnomeUI/Editor.py:1108
msgid "Native Editing"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1137
msgid "Standard Editing"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1264
msgid "mandatory"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1330
msgid "Enter text here"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1382
msgid "New Annotation Name"
msgstr ""

#: ../Pyblio/GnomeUI/Editor.py:1388
msgid "Name of the new annotation:"
msgstr ""

#: ../Pyblio/GnomeUI/Entry.py:114
#, python-format
msgid ""
"Cannot open URL:\n"
"%s"
msgstr ""

#: ../Pyblio/GnomeUI/Fields.py:38
msgid "Authors"
msgstr "Autors"

#: ../Pyblio/GnomeUI/Fields.py:40
msgid "Long Text"
msgstr ""

#: ../Pyblio/GnomeUI/Fields.py:41 ../Pyblio/GnomeUI/glade/config1.glade.h:16
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:21
msgid "URL"
msgstr "URL"

#: ../Pyblio/GnomeUI/Fields.py:42
msgid "Reference"
msgstr "Referéncia"

#: ../Pyblio/GnomeUI/Fields.py:43
msgid "Date"
msgstr "Data"

#: ../Pyblio/GnomeUI/Fields.py:124 ../Pyblio/GnomeUI/glade/config1.glade.h:11
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:10
msgid "Name"
msgstr "Nom"

#: ../Pyblio/GnomeUI/Fields.py:127 ../Pyblio/GnomeUI/glade/config1.glade.h:15
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:20
msgid "Type"
msgstr "Tipe"

#: ../Pyblio/GnomeUI/Fields.py:300
msgid "Available"
msgstr "Disponible"

#: ../Pyblio/GnomeUI/Fields.py:310
msgid "Associated"
msgstr ""

#: ../Pyblio/GnomeUI/Fields.py:321
msgid "Please, select an entry type from previous page."
msgstr ""

#: ../Pyblio/GnomeUI/Fields.py:332
#, python-format
msgid "Fields associated with <b>%s</b> entry type"
msgstr ""

#: ../Pyblio/GnomeUI/FileSelector.py:39
msgid "File"
msgstr "Fichièr"

#: ../Pyblio/GnomeUI/FileSelector.py:76
#: ../Pyblio/GnomeUI/glade/openurl.glade.h:2
msgid "Bibliography type:"
msgstr ""

#: ../Pyblio/GnomeUI/FileSelector.py:93 ../Pyblio/GnomeUI/OpenURL.py:61
msgid " - According to file suffix - "
msgstr ""

#. The root of the search tree is the full database
#: ../Pyblio/GnomeUI/Search.py:71
msgid "Full database"
msgstr ""

#: ../Pyblio/GnomeUI/Search.py:142
msgid "internal error during evaluation"
msgstr ""

#: ../Pyblio/GnomeUI/Search.py:196
#, python-format
msgid ""
"while compiling %s\n"
"error: %s"
msgstr ""

#. These are additional search fields
#: ../Pyblio/GnomeUI/Sort.py:97
msgid "[Entry Type]"
msgstr ""

#: ../Pyblio/GnomeUI/Sort.py:98
msgid "[Key Value]"
msgstr ""

#: ../Pyblio/GnomeUI/Utils.py:193
msgid "The following errors occured:\n"
msgstr ""

#: ../Pyblio/GnomeUI/__init__.py:45
#, python-format
msgid "This is Pybliographic %s [Python %s, Gtk %s, PyGTK %s]"
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:1
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:1
#: ../Pyblio/GnomeUI/glade/medline.glade.h:1
#: ../Pyblio/GnomeUI/glade/openurl.glade.h:1
#: ../Pyblio/GnomeUI/glade/search.glade.h:1
msgid "*"
msgstr "*"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:2
msgid "<b>Entry type</b>"
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:3
msgid "A"
msgstr "A"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:4
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:2
msgid "Association"
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:5
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:3
msgid "Datum"
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:6
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:4
msgid "Editor"
msgstr "Editor"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:7
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:5
msgid "Entries"
msgstr "Intradas"

#: ../Pyblio/GnomeUI/glade/config1.glade.h:9
msgid "FIELDS DEFINITION"
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:10
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:8
msgid "Fields "
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:12
#: ../Pyblio/GnomeUI/glade/fields1.glade.h:11
msgid "Name "
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:13
msgid "Pybliographer Configuration"
msgstr ""

#: ../Pyblio/GnomeUI/glade/config1.glade.h:17
msgid "standard fields"
msgstr ""

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:7
msgid "Entry types and field names configuration"
msgstr ""

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:9
msgid "Information for <b>Article</b> entry type"
msgstr ""

#: ../Pyblio/GnomeUI/glade/fields1.glade.h:12
msgid ""
"Select an entry type and specify \n"
"its associated (mandatory and\n"
"optional) fields on the next page.\n"
"\n"
"\n"
"\n"
msgstr ""

#: ../Pyblio/GnomeUI/glade/format.glade.h:1
msgid "Bibliography style:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/format.glade.h:2
msgid "Format entries"
msgstr ""

#: ../Pyblio/GnomeUI/glade/format.glade.h:3
msgid "Output file:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/format.glade.h:4
msgid "Output format:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/format.glade.h:5
msgid "Save Formatted as"
msgstr ""

#: ../Pyblio/GnomeUI/glade/format.glade.h:6
msgid "Style Selection"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:2
msgid "Enter your Medline Query"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:3
msgid "From:"
msgstr "Expedidor :"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:4
msgid "Limited to:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:5
msgid ""
"Maximum number\n"
"of results:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:7
msgid "Medline Query"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:8
msgid ""
"Only items \n"
"ahead of print"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:10
msgid ""
"Only items \n"
"with abstracts"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:12
msgid ""
"Search \n"
"PubMed for:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:14
msgid ""
"Starting listing at\n"
"result number:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/medline.glade.h:16
msgid "To:"
msgstr "Destinatari :"

#: ../Pyblio/GnomeUI/glade/medline.glade.h:17
msgid ""
"Use the format YYYY/MM/DD\n"
"Month and day are optional."
msgstr ""

#: ../Pyblio/GnomeUI/glade/openurl.glade.h:3
msgid "Enter the location (URI) of the file you would like to open:"
msgstr ""

#: ../Pyblio/GnomeUI/glade/openurl.glade.h:4
msgid "Open Location"
msgstr "Dobrir un emplaçament"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:1
msgid "Cite"
msgstr ""

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:3
msgid "Contents"
msgstr "Somari"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:9
msgid "Open File"
msgstr "Dobrir un fichièr"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:10
msgid "Previous Documents"
msgstr "Documents precedents"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:11
msgid "Pybliographer"
msgstr ""

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:13
msgid "Save File"
msgstr "Enregistrar lo fichièr"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:14
msgid "Settings"
msgstr "Paramètres"

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:15
msgid "_Add..."
msgstr "_Apondre..."

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:16
msgid "_Delete..."
msgstr "_Suprimir..."

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:17
msgid "_Edit..."
msgstr "_Edicion..."

#: ../Pyblio/GnomeUI/glade/pyblio.glade.in.h:18
msgid "_New"
msgstr "_Novèl"

#: ../Pyblio/GnomeUI/glade/search.glade.h:2
msgid "Close"
msgstr "Tampar"

#: ../Pyblio/GnomeUI/glade/search.glade.h:3
msgid "Expert Search"
msgstr ""

#: ../Pyblio/GnomeUI/glade/search.glade.h:4
msgid "Field"
msgstr "Zòna"

#: ../Pyblio/GnomeUI/glade/search.glade.h:5
msgid "Pattern"
msgstr "Motiu"

#: ../Pyblio/GnomeUI/glade/search.glade.h:6
msgid "Query"
msgstr "Consulta"

#: ../Pyblio/GnomeUI/glade/search.glade.h:7
msgid "Search"
msgstr "Recercar"

#: ../Pyblio/GnomeUI/glade/search.glade.h:8
msgid "Simple Search"
msgstr ""

#: ../Pyblio/GnomeUI/glade/sort.glade.h:1
msgid "Select sort criterions"
msgstr ""

#: ../Pyblio/GnomeUI/glade/sort.glade.h:2
msgid "Set as default"
msgstr ""

#: ../Pyblio/LyX.py:37
#, python-format
msgid "no input pipe `%s'"
msgstr ""

#: ../Pyblio/LyX.py:43
#, python-format
msgid "no output pipe `%s'"
msgstr ""

#: ../Pyblio/Open.py:96
#, python-format
msgid "method `%s' provides no opener"
msgstr ""

#: ../Pyblio/Open.py:104 ../Pyblio/Open.py:151
#, python-format
msgid "File `%s' does not exist"
msgstr ""

#: ../Pyblio/Open.py:115 ../Pyblio/Open.py:162
#, python-format
msgid "don't know how to open `%s'"
msgstr ""

#: ../Pyblio/Open.py:143
#, python-format
msgid "method `%s' provides no iterator"
msgstr ""

#: ../Pyblio/Resource.py:77
#, python-format
msgid ""
"Warning: This URL is marked as Invalid or Approximate: %s\n"
"Continue?"
msgstr ""

#: ../Pyblio/Resource.py:81
msgid "Determining Mime Type ... "
msgstr ""

#: ../Pyblio/Resource.py:86
#, python-format
msgid "Cannot determine mime type for item %s "
msgstr ""

#: ../Pyblio/Resource.py:87
#, python-format
msgid ""
"URL in question is: %s\n"
"You should check the url or path given for errors.\n"
"Details: %s"
msgstr ""

#: ../Pyblio/Resource.py:98
msgid "Accessing resource ..."
msgstr ""

#: ../Pyblio/Resource.py:116
#, python-format
msgid "IOError for item %s: cannot uncompress resource."
msgstr ""

#: ../Pyblio/Resource.py:117
#, python-format
msgid ""
"URL: %s\n"
"Details: %s"
msgstr ""

#: ../Pyblio/Resource.py:137
msgid "Starting application ..."
msgstr ""

#: ../Pyblio/Resource.py:140
msgid "No application to view resource"
msgstr ""

#: ../Pyblio/Resource.py:141
#, python-format
msgid ""
"For mime type %s, no entry found in \n"
"configuration option resource/viewers.\n"
"Please consider adding one.\n"
"URL: %s"
msgstr ""

#: ../Pyblio/Style/Parser.py:261
#, python-format
msgid "missing '%s' attribute"
msgstr ""

#: ../Pyblio/Style/Parser.py:271
#, python-format
msgid "invalid opening tag: %s"
msgstr ""

#: ../Pyblio/Style/Parser.py:281
#, python-format
msgid "invalid closing tag: %s"
msgstr ""

#: ../pybliographer.py:43
#, python-format
msgid "This is %s, version %s"
msgstr ""

#: ../pybliographer.py:47 ../pybliographer.py:54
msgid "This is free software with ABSOLUTELY NO WARRANTY."
msgstr ""

#: ../pybliographer.py:48
msgid "For details, type `warranty'."
msgstr ""

#: ../pybliographer.py:111 ../pybliographer.py:163
#, python-format
msgid "%s: error: can't open file `%s'"
msgstr ""

#: ../pybliographer.py:124
#, python-format
msgid "For help, run %s and type `help' at the prompt"
msgstr ""

#: ../pybliographer.py:133
msgid ""
"Useful commands:\n"
"\thelp     to get some help\n"
"\tquit     to quit\n"
msgstr ""

#. save history
#: ../pybliographic.desktop.in.h:1
msgid "Bibliography Manager"
msgstr ""

#: ../pybliographic.desktop.in.h:2
msgid "Manage bibliographic databases"
msgstr ""

#: ../pybliographic.desktop.in.h:3
msgid "Pybliographic Bibliography Manager"
msgstr ""

#: ../scripts/pybcheck.py:31
msgid "usage: pybcheck <file | directory>..."
msgstr ""

#: ../scripts/pybcheck.py:60
#, python-format
msgid "file `%s' is ok [%d entries]"
msgstr ""

#: ../scripts/pybcompact.py:30
msgid "usage: pybcompact <latexfile> <bibtexfiles...>"
msgstr ""

#: ../scripts/pybcompact.py:34
#, python-format
msgid "pybcompact: error: %s\n"
msgstr ""

#: ../scripts/pybcompact.py:99
msgid "no entry"
msgstr ""

#: ../scripts/pybcompact.py:162 ../scripts/pybtext.py:240
#, python-format
msgid "can't find the following entries: %s"
msgstr ""

#: ../scripts/pybconvert.py:29
msgid "usage: pybconvert <source>..<target> <input> [output]"
msgstr ""

#: ../scripts/pybconvert.py:38
msgid "pybconvert: error: bad conversion format"
msgstr ""

#: ../scripts/pybformat.py:34
msgid ""
"usage: pybformat [options] <database...>\n"
"\n"
"    options:\n"
"      -o file, --output=file\t\tspecify an output filename\n"
"      -s style, --style=style\t\tspecify a bibliography style\n"
"      -f format, --format=format\tspecify an output format\n"
"      -H header, --header=header\tdefines a header file\n"
"      -F footer, --footer=footer\tdefines a footer file\n"
"      -l 'output', --list='output'\tlists the available output formats\n"
"      -h, --help\t\t\tshow this help message\n"
"      -v, --version\t\t\tshow this help message\n"
"    "
msgstr ""

#: ../scripts/pybformat.py:49
#, python-format
msgid "pybformat: error: %s\n"
msgstr ""

#: ../scripts/pybformat.py:55
#, python-format
msgid "pybformat: warning: %s\n"
msgstr ""

#: ../scripts/pybformat.py:90 ../scripts/pybtext.py:252
#: ../scripts/pybtext.py:278 ../scripts/pybtext.py:321
#, python-format
msgid "can't open `%s': %s"
msgstr ""

#: ../scripts/pybformat.py:97
#, python-format
msgid "unknown list `%s'"
msgstr ""

#: ../scripts/pybformat.py:100
#, python-format
msgid "pybformat: available values for `%s':"
msgstr ""

#: ../scripts/pybformat.py:104
#, python-format
msgid "empty value list `%s'"
msgstr ""

#: ../scripts/pybformat.py:132
msgid "too few arguments"
msgstr ""

#: ../scripts/pybformat.py:139 ../scripts/pybtext.py:117
#, python-format
msgid "unknown output format `%s'"
msgstr ""

#: ../scripts/pybformat.py:152 ../scripts/pybtex.py:86
#: ../scripts/pybtext.py:94
#, python-format
msgid "can't find style `%s'"
msgstr ""

#: ../scripts/pybformat.py:155
#, python-format
msgid "pybformat: using style `%s', format `%s'\n"
msgstr ""

#: ../scripts/pybformat.py:170
#, python-format
msgid "can't open header `%s': %s"
msgstr ""

#: ../scripts/pybformat.py:178
#, python-format
msgid "can't open database: %s"
msgstr ""

#: ../scripts/pybformat.py:193
#, python-format
msgid "can't open footer `%s': %s"
msgstr ""

#: ../scripts/pybtex.py:34
msgid "usage: pybtex <latexfile> [bibtexfiles...]"
msgstr ""

#: ../scripts/pybtex.py:38
#, python-format
msgid "pybtex: error: %s\n"
msgstr ""

#. warn the user that some entries were not found
#: ../scripts/pybtex.py:59
msgid "pybtex: warning: the following keys were not resolved"
msgstr ""

#. If the LaTeX document declares no style...
#: ../scripts/pybtex.py:64
msgid "no style defined"
msgstr ""

#: ../scripts/pybtext.py:41
msgid "usage: pybtext [-o outputfile] [-s style] <textfile> <bibfiles...>"
msgstr ""

#: ../scripts/pybtext.py:45
#, python-format
msgid "pybtext: error: %s\n"
msgstr ""

#: ../scripts/pybtext.py:50
#, python-format
msgid "pybtext: warning: %s\n"
msgstr ""

#: ../scripts/pybtext.py:124
#, python-format
msgid "File already exists: `%s'"
msgstr ""

#: ../scripts/pybtext.py:127
#, python-format
msgid "A file with the same name already exists: `%s'"
msgstr ""

#: ../scripts/pybtext.py:178
msgid "no citation found"
msgstr ""

#: ../scripts/pybtext.py:193
#, python-format
msgid "pybtext: using style `%s', format `%s'\n"
msgstr ""

#: ../scripts/pybtext.py:316
#, python-format
msgid "can't create `%s'"
msgstr ""

#: ../scripts/pybtext.py:342
#, python-format
msgid "can't remove `%s'"
msgstr ""

#: ../scripts/pybtext.py:344
msgid "Done"
msgstr "Terminat"

